README.md

```diff
@@ -1,43 +1,62 @@
-
-Here is an example:
+<br/>**The `mod.json` format:**
 ```jsonc
 {
 	"lastUpdate": "2024-01-01T00:00:00Z",
 	"url": "https://example.com/path/to/your/mod.zip",
 	"source": "https://example.com/path/to/the/source/code",
-	"sha256": "The sha256 hash of your zip file"
+	"sha256": "The sha256 hash of your zip file",
+	"icon": "https://example.com/path/to/myicon.webp"
 }
 ```
-- `tags` - Optional parameter. An array of strings. See below for more information.
+
+<br>**The fields:**
+- `tags (optional)` - An array of strings that contain information about your mod - see below.
 - `lastUpdate` - When the last update to this mod. Every time you update it, The date format is `YYYY-MM-DD`, along with the `T00:00:00Z` string at the end.
 - `url` - The URL to your ZIP file, which will be automatically downloaded and a copy will be hosted on the website.
-- `source` - Optional parameter (but recommended). A link to the source code of the mod.
-- `sha256` The sha256 hash of the ZIP file. The hash of the retrieved file in `url` must match this.
-
-
-If your mod contains the output of generative AI, defined here as LLM-generated text, artwork/images, music/sounds, or assets *other than code*, include the optional `tags` parameter, and add the `gen-ai` tag (If your mod contains LLM-generated code, simply let us know in your PR):
-
+- `source (optional)` - A link to the source code of the mod. Ideally, hosted on either [Github](https://www.github.com) or [Codeberg](https://www.codeberg.org).
+- `sha256` - The sha256 hash of the ZIP file. The hash of the retrieved file in `url` must match this. (You can get this by running `Get-FileHash <path>` on Windows or `sha256sum <path>` on Linux.)
+- `icon (optional)` - A URL to an image to be displayed for your mod where applicable. Please keep it a multiple of 256x256, small, and webp/png/jpeg.
+
+<br/>**Tagging your mod:**<br/>
+Tags are a quick way to organise mods into various categories. **They are optional, except for gen-ai. (if applicable)** Here are some tags that you can apply:
+- `gen-ai` - If your mod contains the output of generative AI, defined here as LLM-generated text, artwork/images, music/sounds, or assets *other than code*, **add this tag.** If your mod contains LLM-generated code, simply let us know in your PR.
+- `content` - Mods that extend the gameplay of the game (for example, adding new dialogue or interactions).
+- `dev` - Mods that are intended for developers/to aid in the development in content surrounding the game.
+- `qol` - Mods intended for quality of life features (for example, bug fixes or accessibility features).
+- `lib` - Mods intended to be a shared code library for other mods.
+- `mature` - Mods that contain more mature themes that require viewer discretion.
+
+<br>**Here is how you would apply tags:**
 ```jsonc
 {
-	"tags": ["gen-ai"],
+	"tags": ["gen-ai", "content"],
 	// The rest of the JSON
 }
 ```
+<br/>**And to use the repository as an API:**<br>
```

Comment:
I feel like this should be moved down or put elsewhere (maybe under a new section?), since otherwise it feels like it could be confusing for mod devs.